config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  root "posts#index"
```

Comment:
I forgot to do a git add all be right back. Thanks for the suggestion cause I somehow missed that part.